app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<section class="recipes">
+  <% @recipes[:results].each do |recipe| %>
```

Comment:
Probably a good place for a refactor here so we're not having to pass hashes into the view

Reply:
It's definitely doable but I was kinda curious whether or not that belonged in the front-end.. Is it a model? A poro? A facade? A serializer?